packages/blunderbuss/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "@octokit/webhooks": "^7.24.2",
+    "@octokit/webhooks-definitions": "^3.57.2",
```

Comment:
huh - is this used anywhere?

Reply:
I think it's needed when pulling in the `EventPayloads` type from octokit/webhooks.